scraper/src/mindtouch2zim/processor.py

```diff
@@ -475,11 +475,12 @@ def _process_page(
             post_head_insert=None,
             notify_js_module=None,
         )
-        if (
-            self.mindtouch_client.library_url.endswith(".libretexts.org")
-            and page.title == "Glossary"
+        if self.mindtouch_client.library_url.endswith(".libretexts.org") and re.match(
```

Comment:
Can you add a comment with an example path. Will be very helpful to know what the code was working with when should it change in the future